core/src/main/java/io/questdb/cairo/sql/async/PageFrameSequence.java

```diff
@@ -63,7 +63,7 @@ public class PageFrameSequence<T extends StatefulAtom> implements Closeable {
     private final AtomicBoolean valid = new AtomicBoolean(true);
     private final WorkStealingStrategy workStealingStrategy;
     public volatile boolean done;
-    private SqlExecutionCircuitBreaker circuitBreaker;
+    private SqlExecutionCircuitBreakerWrapper circuitBreaker;
```

Comment:
nit: we can now change `circuitBreaker = Misc.freeIfCloseable(circuitBreaker);` to `circuitBreaker = Misc.free(circuitBreaker);` in the `close()` method.

Reply:
will change it

Reply:
done